### PR TITLE
Scorecard poc ingestor React Frontend

### PR DESCRIPTION
**Pull request by lucamolteni**
React front-end for https://github.com/kiegroup/akrivis/issues/6

### CODE REVIEW

File: plugins/rules-backend/src/service/ScoreCardRunner.ts [hunk 1]
```diff
@@ -18,10 +18,77 @@ export interface Threshold {
...
   return resp.data;
 }
+
+export async function runJobs(): Promise<Record[]> {
+  const url = 'http://localhost:8082/job';
+
+  const resp = await axios.get(url);
+
+  return resp.data;
+}
```
Comment (Rikkola) [inline plugins/rules-backend/src/service/ScoreCardRunner.ts:41]: I will see if I can make these round trips prettier.

Comment (lucamolteni) [reply plugins/rules-backend/src/service/ScoreCardRunner.ts:41]: TBF I'm not even sure we need these. we should perhaps check with backstage devs if this architecture is a must-have

Comment (Rikkola) [reply plugins/rules-backend/src/service/ScoreCardRunner.ts:41]: It is. The client can only connect to the backend or browser assumes cross site scripting is in place and prevents the connection. 
But looking at the manual a proxy is easy to setup. Trying it out. 
https://backstage.io/docs/plugins/call-existing-api/#using-the-backstage-proxy

Comment (lucamolteni) [reply plugins/rules-backend/src/service/ScoreCardRunner.ts:41]: https://github.com/kiegroup/akrivis/issues/9

**Pull request merged**